components.yaml

```diff
@@ -1,22 +1,29 @@
 components:
-  multus:
+  - name: "multus"
```

Comment:
The API needs to remain the same, it is a public API. Is there an issue with adapting the existing structure where key is the name of the component?

Reply:
The reason of the change is mainly because if I parse in the original format then I loose the component name, since it doesn't have a value.
Also, It was more elegent to go over the components as an array of the same struct (that also holds the name).

Reply:
You can treat `components` as a `map[string]component` and be able to keep this. Would that help?

Reply:
Having the name as a part of an item in a list is more Kubernetes way of doing API. However, this API is already consumed externally, let's not break it.

Reply:
please note that  I'll have to change the parameters names, because for example `bridge-marker` does not unmarshal well, so I'll use `bridgemarker`. 
So I think will still need to do some adjustments to the midstream script.

Reply:
Python yaml library was able to to cope with that 😬 could we keep the name in quotes then?

Reply:
yay found how to make it work with the original keys.